ansible/my-ansible-playbook.yml

```
@@ -0,0 +1,67 @@
+- name: Exercise 4 Playbook - Setup Web Servers
+  hosts: managed_nodes
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_api_key
+    prompt: What is your TRELLO API KEY?
+
+  - name: trello_api_token
+    prompt: What is your TRELLO API TOKEN?
+
+  tasks:
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: present
+    become: yes
+
+  - name: Install Python3
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+    become: yes
+    
+  - name: Install Poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+    args: 
+      creates: /home/ec2-user/.local/bin/poetry
+
+  - name: Create Todo-app Directory
+    ansible.builtin.shell: mkdir /opt/todoapp
+    args:
+      creates: /opt/todoapp
+    become: yes
+
+  - name: Change Todo-app Folder Owner to ec2-user
+    ansible.builtin.shell: chown ec2-user /opt/todoapp
+    become: yes
```

Minor, but worth noting that we can use the builtin file module to do this in one step, and personally more importantly than being more concise it might handle more edge cases than we've considered (e.g. easy to adjust to also set explicit permissions). Untested but something like below:
```suggestion
  - name: Create Todo-app Directory
    ansible.builtin.file:
      path: /opt/todoapp
      owner: ec2-user
      state: directory
    become: yes
```